sites/default/menu_data.json

```diff
@@ -988,20 +988,6 @@
         "target" : "pop",
         "children" : [ ],
         "requirement" : 1
-    }, {
-        "label" : "Export",
-        "menu_id" : "Popup:Export",
-        "url" : "/interface/../custom/export_xml.php",
-        "target" : "pop",
-        "children" : [ ],
-        "requirement" : 1
-    }, {
```

Comment:
@bauripalash great this does it. With the files no longer there, no need for the menu
![screen shot 2018-12-07 at 2 46 18 am](https://user-images.githubusercontent.com/16350814/49622572-62bc0780-f9ca-11e8-8fed-d0e3ffdf6df9.png)

Reply:
Thanks to @tmccormi for suggesting me this